packages/cli/src/aws/init.ts

```diff
@@ -233,18 +239,21 @@ export async function initStackCommand(): Promise<void> {
   print('These values will be encrypted at rest.');
   print(`The values will be stored in the "/medplum/${config.name}" path.`);
 
-  const serverParams = {
+  const serverParams: Record<string, string | number> = {
```

Comment:
Can this be a `Partial`?
```suggestion
  const serverParams: Partial<MedplumInfraConfig> = {
```